internal/golden/golden.go

```diff
@@ -36,11 +37,11 @@ type goldenOptions struct {
 	path string
 }
 
-// GoldenOption is a supported option reference to change the golden files comparison.
-type GoldenOption func(*goldenOptions)
+// Option is a supported option reference to change the golden files comparison.
+type Option func(*goldenOptions)
```

Comment:
Probably rename the `goldenOptions` type to `options` also?

Reply:
I renamed it to `config` to avoid name collisions